samples/bluetooth/direct_test_mode/src/dtm_hw_config.h

```diff
@@ -11,8 +11,9 @@
 extern "C" {
 #endif
 
-#if defined(NRF5340_XXAA_NETWORK) || defined(NRF52833_XXAA) || \
-	defined(NRF52811_XXAA) || defined(NRF52820_XXAA)
+#if (defined(NRF5340_XXAA_NETWORK) || defined(NRF52833_XXAA) || \
```

Comment:
I think it could be just `#if DT_PROP(DT_PATH(zephyr_user), dtm_antenna_count) > 0`

Reply:
Cannot anyone define a zephyr,user area? Ie. that isn't guarded against actual hardware capabilities?

Reply:
Instead of checking the specific boards you can use `CONFIG_HAS_HW_NRF_RADIO_BLE_DF`, similarly the one below could be `HAS_HW_NRF_RADIO_BLE_CODED`

Reply:
There's no specific define or selection that is currently available for checking AoA/AoD capable SoC's, atleast that I could find. I believe such a feature is outside of the scope of this PR.